src/pytest_docker/plugin.py

```diff
@@ -119,6 +142,27 @@ def wait_until_responsive(
 
         raise Exception("Timeout reached while waiting on service!")
 
+    def display_live_logs(self, service: str) -> None:
+        if service in self._live_logs:
+            return
+        self._live_logs[service] = self._thread_pool_executor.submit(
+            self._docker_compose.execute, f"logs {service} -f"
+        )
```

Comment:
The thread pool executor has max_worker limit set dependning on user's machine, potentially resulting in the code "not doing anything" here because the pool has no free worker - perhaps set the max_worker parameter in ThreadPoolExecutor based on `self._live_logs`?

Reply:
Nice catch! What do you think of specifying a big enough value, e.g. 100? We don't know beforehand how many `display_live_logs` calls we'll receive, and we can't change the `_max_workers` value because it is private.

Reply:
Yeah, I think that should be fine, 100 _ought to be enough for everyone_, and it's upper limit only, so it spawn only as many threads as it needs.